src/lib/core/evio.c

```diff
@@ -294,69 +324,64 @@ evio_service_bind_addr(struct evio_service *service)
  *
  * @retval 0 for success
  */
-int
-evio_service_listen(struct evio_service *service)
+static int
+evio_service_entry_listen(struct evio_service_entry *entry)
```

Comment:
I propose to call it `..._create()`. `init` is a wrong suffix which we have a chance to fix here alongside in all places since these lines are changed anyway.